### PR TITLE
Additional grafana dashboards

### PR DESCRIPTION
**Pull request by coltoneshaw**
#### Summary
Adding the additional grafana dashboards that have come out of the performance investigation.

### CODE REVIEW

File: source/scale/performance-monitoring.rst [hunk 1]
```diff
@@ -112,9 +112,9 @@ Getting Started
 ---------------
 
 To help you get started, you can download three sample dashboards shared in Grafana:
-
+- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`__, which contains detailed charts for performance monitoring including applicataion, cluster, job server, and system metrics.
+- `Mattermost Collapsed Reply Threads Metrics <https://grafana.com/grafana/dashboards/15581>`__, wwhich contains detailed metrics on the queries involved in our **beta** collapsed reply threads feature.
```
Comment (cwarnermm) [inline source/scale/performance-monitoring.rst:116]:
```suggestion
- `Mattermost Collapsed Reply Threads Metrics <https://grafana.com/grafana/dashboards/15581>`__, which contains detailed metrics on the queries involved in our Collapsed Reply Threads Beta feature.
```

File: source/scale/performance-monitoring.rst [hunk 2]
```diff
@@ -112,9 +112,9 @@ Getting Started
 ---------------
 
 To help you get started, you can download three sample dashboards shared in Grafana:
-
+- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`__, which contains detailed charts for performance monitoring including applicataion, cluster, job server, and system metrics.
```
Comment (cwarnermm) [inline source/scale/performance-monitoring.rst:115]:
```suggestion
- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`__, which contains detailed charts for performance monitoring including application, cluster, job server, and system metrics.
```

**Review by cwarnermm — COMMENTED** [review]
Thank you, @coltoneshaw, for this important docs update! I've provided suggestions inline.

File: source/scale/performance-monitoring.rst (hunk 2 above)
Comment (coltoneshaw) [reply source/scale/performance-monitoring.rst:115]: I'm ashamed at my spelling in this...

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**